packages/core/src/constants.ts

```diff
@@ -31,7 +31,8 @@ export const APP_INITIAL_STATE: AppState = {
 }
 
 export const STORAGE_KEYS = {
-  TERMS_AGREEMENT: 'onboard.js:agreement'
+  TERMS_AGREEMENT: 'onboard.js:agreement',
+  LAST_CONNECTED_WALLET: 'onboard.js:last_connected_wallet'
```

Comment:
Would we want to update the ls key to be prefixed with `web3-onboard` instead of `onboard.js` for clarity?

Reply:
We could, but then all users would need to re-agree to the terms and conditions....